src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -0,0 +1,191 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import { repeat } from "lit-html/directives/repeat";
+
+import "../../../components/ha-card.js";
+import "../components/hui-entities-toggle.js";
+
+import { fireEvent } from "../../../common/dom/fire_event.js";
+import { DOMAINS_HIDE_MORE_INFO } from "../../../common/const.js";
+import { HassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+import { HomeAssistant } from "../../../types.js";
+import createRowElement from "../common/create-row-element.js";
+import computeDomain from "../../../common/entity/compute_domain.js";
+import processConfigEntities from "../common/process-config-entities";
+
+interface EntityConfig {
+  name?: string;
+  icon?: string;
+  entity: string;
+  type?: string;
+  secondary_info: "entity-id" | "last-changed";
+  action_name?: string;
+  service?: string;
+  service_data?: object;
+  url?: string;
+}
+
+interface Config extends LovelaceConfig {
+  show_header_toggle?: boolean;
+  title?: string;
+  entities: EntityConfig[];
+}
+
+class HuiEntitiesCard extends HassLocalizeLitMixin(LitElement)
+  implements LovelaceCard {
+  protected hass?: HomeAssistant;
+  protected config?: Config;
+  protected configEntities?: EntityConfig[];
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      config: {},
+    };
+  }
+
+  public getCardSize() {
+    // +1 for the header
+    if (!this.config) {
+      return 0;
+    }
+    return this.config.title ? 1 : 0 + this.config.entities.length;
```

Comment:
I don't think it does what you think it does.

Reply:
oh youre right... oops